generator/explores/ping_explore.py

```diff
@@ -15,13 +15,33 @@ class PingExplore(Explore):
 
     def _to_lookml(self, v1_name: Optional[str]) -> List[Dict[str, Any]]:
         """Generate LookML to represent this explore."""
+        views_lookml = self.get_view_lookml(self.views["base_view"])
+        views = [view["name"] for view in views_lookml["views"]]
+
+        joins = []
```

Comment:
Guess we may as well add a type annotation here?

```suggestion
        joins: List[Dict[str, str]] = []
```